pkg/storage/etcd/etcd_test_util.go

```diff
@@ -155,3 +157,10 @@ func NewEtcdTestClientServer(t *testing.T) *EtcdTestServer {
 	}
 	return server
 }
+
+// NewEtcdTestStorage creates a new storage.Interface and TestServer
+func NewEtcdTestStorage(t *testing.T, codec runtime.Codec, prefix string) (*EtcdTestServer, storage.Interface) {
```

Comment:
Correct - I copied it, because I needed it.
But I think your PR will be merged sooner anyway.